Cesium.CodeGen/Ir/BlockItems/FunctionDefinition.cs

```diff
@@ -285,25 +290,351 @@ public void EmitTo(IEmitScope scope)
 
     private void EmitCode(TranslationUnitContext context, FunctionScope scope)
     {
-        var statement = _statement.Lower(scope);
-        statement.EmitTo(scope);
-        if (statement.HasDefiniteReturn == false)
+        var loweredStmt = (CompoundStatement) _statement.Lower(scope);
+
+        var dset = new AdjacencyGraph<CodeBlockVertex, CodeBlockEdge>();
```

Comment:
Whole process IMO should reside between Lower phase and Emit phase there
https://github.com/ForNeVeR/Cesium/blob/b23fe23a9ca71853dc8590413df476176a3cbcf3/Cesium.CodeGen/Contexts/AssemblyContext.cs#L48-L51